src/modules/syncedVideoPlayer/SyncedVideoPlayer.ts

```diff
@@ -0,0 +1,78 @@
+import { Socket } from "socket.io-client";
+
+interface User {
+  id: string;
+  name: string;
+}
+
+class SyncedVideoPlayer {
+
+  /**
+   * User {id, name}
+   */
+  user: User
+
+  /**
+   * videoId - youtube videoId
+   */
+  videoId: String
+
+  /**
+   * channel - communication channel
+   */
+  channel: Socket
+
+  /**
+   * room - connected party socket room name
+   * Note: room and channel are the same thing in socket.io
+   */
+  room: String
+
+  /**
+   * player - Youtube player object
+   */
+  player: any
```

Comment:
Can we avoid `any` here

Reply:
what do you suggest I keep it as?

I don't know what is the object type of the Youtube player object. It's an Object, but I couldn't find any info on a type for just simple object, without defining the actual object as interface.

Reply:
You'll have to create your own interface based on the Youtube API documentation.